### PR TITLE
Create hello-libs sample

### PR DESCRIPTION
**Pull request by ggfan**
Create hello-libs sample project:
       1) one application [uses 2 3rd party libs]
       2) 2 shared libs
       3) lib distribution dir is at ${project}/distribution/...
    
    Tested:
       Mac OS X + Simulator

Please kindly review and let me know your thoughts, thanks!
[Repo CI related thing is not addressed with this code change]

### CODE REVIEW

File: hello-libs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+*.iml
```
Comment (hak) [inline hello-libs/.gitignore]: Don't need the file?

File: hello-libs/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+Hello-libs
+=========
+Hello-Libs is an Android sample that demos native lib management in Android Studio.
+
+This sample uses the new [Gradle Experimental Android plugin](http://tools.android.com/tech-docs/new-build-system/gradle-experimental) with external lib support:  
+*    Build JUST an native shared library[for distribution]
```
Comment (hak) [inline hello-libs/README.md]: nit 'an' -> 'a'

File: hello-libs/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+*    Use 3rd party shared library / Static library   
+
+The key points are:
+*    Inform Android Studio where lib/header is
+*    Add lib dependency to application
+Details are inside projects' build.gradle files
+
+Sample contents:
+*    Create 2 shared libs
+*    Copy builts libs into distribution directory
```
Comment (hak) [inline hello-libs/README.md]: built

File: hello-libs/app/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (hak) [inline hello-libs/app/.gitignore]: don't need the file.

File: hello-libs/.gitignore (hunk 1 above)
Comment (ggfan) [inline hello-libs/.gitignore]: no, this file is generated by android studio

File: hello-libs/README.md (hunk 2 above)
Comment (ggfan) [inline hello-libs/README.md]: true: correct in next upload for sure. thanks

File: hello-libs/build_lib.sh [hunk 5]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+pushd /Users/gfan/proj/android-ndk/hello-libs
+
+gradlew  :gmath:clean :gmath:assemble :gperf:clean :gperf:assemble
+
+mkdir  -p distribution/gmath/lib
```
Comment (hak) [inline hello-libs/build_lib.sh]: Wondering if we should copy files with gradle rather than sh?

File: hello-libs/gmath/src/main/jni/gmath.c [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+#define GMATH_EXPORT
+#endif
+
+#define LOGE(...) \
+  ((void)__android_log_print(ANDROID_LOG_ERROR, "gmath::", __VA_ARGS__))
+
+/*
+ * return 2 ^ n with multiplication implementation
+ */
+GMATH_EXPORT unsigned power(unsigned n) {
```
Comment (hak) [inline hello-libs/gmath/src/main/jni/gmath.c]: I understand the API is just a demonstration purpose.
But power() is in standard lib. Do we have any better example?

File: hello-libs/app/.gitignore (hunk 4 above)
Comment (ggfan) [inline hello-libs/app/.gitignore]: true: removed

File: hello-libs/README.md (hunk 3 above)
Comment (ggfan) [inline hello-libs/README.md]: yes, fixed in next

File: hello-libs/build_lib.sh (hunk 5 above)
Comment (ggfan) [inline hello-libs/build_lib.sh]: deleted: it is not needed. I forgot to remove it. Gradle handles all

File: hello-libs/gmath/src/main/jni/gmath.c (hunk 6 above)
Comment (ggfan) [inline hello-libs/gmath/src/main/jni/gmath.c]: I think pow() is the standard function name. This is just something that simple enough to have something -- like a place holder. The goal is to show gradle's library processing, the content is not that significant.

**Pull request closed without merging**